src/nxdoc/web-ui/web-ui-tutorials/how-to-create-and-customize-nuxeo-actions-menu.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'HOWTO: Create and Customize a Nuxeo actions menu'
+review:
+    comment: ''
+    date: '2020-03-25'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to create and customize a Nuxeo actions menu.
+        level: Advanced
+        tool: code
+        topics: Web UI
+labels:
+    - lts2017-ok
+    - lts2019-ok
+    - tutorial
+    - nuxeo-elements
+    - nuxeo-ui-elements
+    - polymer
+tree_item_index: 500
+---
+
+{{! excerpt}}
+In this tutorial you will learn how to create and customize a Nuxeo actions menu on your code.
```

Comment:
I don't know if people know what a Nuxeo actions menu is. We should explain that it is a responsive menu that can hold actions (e.g. buttons), which will hide some of them behind a dropdown according to the available (or predefined space).